packages/nodejs/src/instrumentation/http/lifecycle/incoming.ts

```diff
@@ -83,6 +86,19 @@ function incomingRequest(
   }
 }
 
+function filterHeaders(headers: IncomingHttpHeaders): HashMap<any> {
+  let filtered: HashMap<any> = {}
+  const headersAllowList = global.__APPSIGNAL__.config.data.requestHeaders || []
+
+  headersAllowList.forEach(key => {
+    if (headers[key] != undefined) {
```

Comment:
```suggestion
    if (headers[key] !== undefined) {
```

Reply:
Thanks to the TS compiler we don't need to write equality operators as in vanilla JS